mkdocs.yml

```
@@ -56,6 +56,11 @@ nav:
       - reference/command-reference/flox.md
       - ... | flat | reference/command-reference/*
 
+validation:
+  omitted_files: warn
+  absolute_links: warn
+  unrecognized_links: warn
+
```

```suggestion
validation:
  omitted_files: info
  absolute_links: info
  unrecognized_links: warn

```

Or we could do this, but that might defeat the whole purpose of this PR?

---

I've turn the validation off only in the nav